modules/functions.js

```diff
@@ -61,7 +61,7 @@ module.exports = (client) => {
     if (text && text.constructor.name == "Promise")
       text = await text;
     if (typeof evaled !== "string")
-      text = require("util").inspect(text, {depth: 0});
+      text = require("util").inspect(text, { depth: 0 });
```

Comment:
Although this is similar to the original code, I highly suggest this to be moved outside functions. [global-require](https://eslint.org/docs/rules/global-require) rule in ESLint.

Reply:
This is what it was from original code, I am gonna update this.